### ISSUE TITLE
Can we configure Authentication on Web GUI

### ISSUE THREAD

**Issue by masifpak** [spender-sandbox/cuckoo-modified#473]
I want authentication to login cuckoo web interface. How can do this.

**Comment by Nagaev** [comment]
You just have to enable the Web Authentification in $CUCKOO/web/web/settings.py
`WEB_AUTHENTIFICATION = True
`You can add accounts through $CUCKOO/web/manage.py

**Comment by masifpak** [comment]
I am facing the following error. 


> django.db.utils.OperationalError: no such table: auth_user

![image](https://user-images.githubusercontent.com/12061167/28619772-804e869a-7223-11e7-96a4-f4fda6528580.png)


Cuckoo is going to sqlite3 while i am using mysql.
I am using the following way to create a user.
sudo  /opt/cuckoo/web/manage.py   createsuperuser

**Comment by masifpak** [comment]
Cuckoo is going to sqlite3 while i am using mysql.

**Comment by Nagaev** [comment]
I'm using sqlite for web auth since I have few accounts to add.

If you want to switch to MySQL, you can edit $CUCKOO/web/web/settings.py with the section DATABASES, refer to the Django documentation for the syntax.

You should have to create manually every table in your mysql db to match the default sqlite db.

**Comment by masifpak** [comment]
Thanks for your reply. Can we use both dbs. mysql for status of tasks and sqlite for authentication.

**Comment by Nagaev** [comment]
Of course, this is a  db dedicated to Web auth and users roles. I'm using Postgres and Sqlite. Actually the web auth DB (sqlite db located in $CUCKOO/web) has nothing to do with the Cuckoo engine. You can change it but I think sqlite is sufficient to manage web auth.

**Comment by masifpak** [comment]
Then where to set sqlite db for authenticating users only and why is it missing auth_user table .

**Comment by masifpak** [comment]
I have a service which submit malwares using API  like as following, Will it also need credentials to submit malwares or we can exclude this API from authentication.

> sudo curl -F file=@/home/itsec/file.exe  http://172.20.16.97:8080/api/tasks/create/file/

**Comment by masifpak** [comment]
Sync your DB and set username and password during this synching. 

./manage.py syncdb
